functions/Modules.ts

```diff
@@ -0,0 +1,22 @@
+export class Game { 
+    //Fields 
+    gameName: string; 
+    yearReleased?: number;
+    genre?: string;
+    console?: string;
+    developer?: string;
+  
+    //Constructor 
+    constructor(gameName:string, yearReleased?:number, genre?:string, console?:string, developer?:string) { 
```

Comment:
Not a big deal right now but in the future you'll want to put more limitations on the inputs to your games. Character length, properly formatted years, etc.